### ISSUE TITLE
Questionable real-time search results

### ISSUE THREAD

**Issue by mperham** [rubyapi/rubyapi#145]
Hi I saw RubyAPI this morning and thought I would test it out by searching for Thread.handle_interrupt.  `handle` did not show it nor did `handle_`.  `handle_inter` shows no results at all.  Only once I type `handle_interrup` does it actually show up.  Seems like this might be something you could tune, I hope this feedback is helpful!

**Comment by colby-swandale** [comment]
👋 Thanks for the feedback! It's defiantly super helpful on what areas needs improving. The autocomplete is defiantly an area that hasn't been optimised at all 😅, but It's on my priority list of things to improve.

**Comment by colby-swandale** [comment]
Hi! I have updated the autocomplete engine to return much more accurate results. Typing `handle` in the search will now show `Thread::handle_interrupt` as a result. 

Thank you for your feedback and helping improve the user experience!

**Comment by tjwallace** [comment]
Not sure if I should put this here or in a new issue but when I search for `set` I'd expect to find `Set` (class) in the first few results but it is not there:
![image](https://user-images.githubusercontent.com/91084/65621420-90356a00-df78-11e9-8d8f-13d3d3d43067.png)

When I search for `Array` I get better results:
![image](https://user-images.githubusercontent.com/91084/65621389-8449a800-df78-11e9-9a1e-a6b43cbce3fb.png)

**Comment by colby-swandale** [comment]
@tjwallace thanks for reporting this! `Set` should defiantly be the first result. Would you be able to create a new ticket with this information?

**Comment by tjwallace** [comment]
@colby-swandale no problem: #193
